### PR TITLE
docs: added upgrade instructions

### LINKED ISSUE #1381: Add upgrade instructions to README.md
We don't have instructions on how to upgrade in the cli repo readme nor in the changelogs. It might be a good idea to have a thing in both places.

`pip install instructlab --upgrade` worked for me, but i dont know if i have to upgrade venv as well as a good practice or whatnot

### PR DESCRIPTION
**Pull request by sanskriti2005**
**Issue resolved by this Pull Request:**
Resolves #1381 

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the
  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] [Changelog](https://github.com/instructlab/instructlab/blob/main/CHANGELOG.md) updated with breaking and/or notable changes for the next minor release.
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
     - [Train the model in the cloud](#train-the-model-in-the-cloud)
   - [📜 Test the newly trained model](#-test-the-newly-trained-model)
   - [🍴 Serve the newly trained model](#-serve-the-newly-trained-model)
+- [🤩 Install latest instructLab upgrades](#-upgrade-instructlab-to-latest-version)
```
Comment (leseb) [inline README.md:35]: How about using an 🚀 for representing upgrades?

File: README.md [hunk 2]
```diff
@@ -566,6 +567,13 @@ argument to specify your new model:
    and exists in a directory with the suffix `trained`. For example:
    `instructlab-merlinite-7b-lab-trained/instructlab-merlinite-7b-lab-Q4_K_M.gguf`.
 
+## 🤩 Upgrade instructLab to latest version
```
Comment (leseb) [inline README.md:570]: ditto

**Review by leseb — CHANGES_REQUESTED** [review]
Please update your commit message to add a sign-off, thanks!

File: README.md (hunk 1 above)
Comment (sanskriti2005) [reply README.md:35]: Sure, it looks better.

File: README.md [hunk 3]
```diff
@@ -32,6 +32,7 @@
     - [Train the model in the cloud](#train-the-model-in-the-cloud)
   - [📜 Test the newly trained model](#-test-the-newly-trained-model)
   - [🍴 Serve the newly trained model](#-serve-the-newly-trained-model)
+- [🚀 Install latest instructLab upgrades](#-upgrade-instructlab-to-latest-version)
```
Comment (kelbrown20) [inline README.md:35]: Hello, thank you for this great work! Can we change this title in the toc so its consistent with the section title? I like the "Upgrade instructLab to latest version"

Can we also move it so its after the "Chat with the new model (not optional this time)" section. I think that would be better so the workflow is still consistent.

Comment (kelbrown20) [reply README.md:35]: Also ensure the capitalization are consistent
instructlab  -> InstructLab

File: README.md [hunk 4]
```diff
@@ -566,6 +567,13 @@ argument to specify your new model:
    and exists in a directory with the suffix `trained`. For example:
    `instructlab-merlinite-7b-lab-trained/instructlab-merlinite-7b-lab-Q4_K_M.gguf`.
 
+## 🚀 Upgrade instructLab to latest version
```
Comment (kelbrown20) [inline README.md:570]: Same here with the caps

File: README.md (hunk 3 above)
Comment (sanskriti2005) [reply README.md:35]: Sure! Thanks for the review

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]
@sanskriti2005 I will approve once commits are squashed/CI failures are fixed

**Comment by sanskriti2005** [comment]
@nathan-weinberg I've done as you instructed. :)

File: README.md [hunk 5]
```diff
@@ -33,6 +33,7 @@
   - [📜 Test the newly trained model](#-test-the-newly-trained-model)
   - [🍴 Serve the newly trained model](#-serve-the-newly-trained-model)
 - [📣 Chat with the new model (not optional this time)](#-chat-with-the-new-model-not-optional-this-time)
+- [🚀 Upgrade InstructLabs to latest version](#-upgrade-instructlab-to-latest-version)
```
Comment (leseb) [inline README.md:36]:
```suggestion
- [🚀 Upgrade InstructLab to latest version](#-upgrade-instructlab-to-latest-version)
```

File: README.md [hunk 6]
```diff
@@ -576,6 +577,13 @@ argument to specify your new model:
 
    If you are interested in optimizing the quality of the model's responses, please see [`TROUBLESHOOTING.md`](./TROUBLESHOOTING.md#model-fine-tuning-and-response-optimization)
 
+## 🚀 Upgrade InstructLab to latest version
+- To upgrade InstructLab to the latest version, use the following command:
+
+   ```sh
```
Comment (leseb) [inline README.md:583]:
```suggestion
   ```shell
```

**Review by leseb — COMMENTED** [review]
nits

**Comment by nathan-weinberg** [comment]
@sanskriti2005 why was this closed?

**Pull request closed without merging**